.github/workflows/release.yml

```
@@ -47,12 +47,16 @@ jobs:
           title: 'chore(release): ${{ steps.extractver.outputs.version }}'
           body: 'Version bump in package.json and package-lock.json for release [${{ steps.extractver.outputs.version }}](https://github.com/${{github.repository}}/releases/tag/v${{ steps.extractver.outputs.version }})'
           branch: version-bump/${{ steps.extractver.outputs.version }}
+        # Version check based on package.json version number
+      - name: Check if version changed
+        uses: EndBug/version-check@v1.6.0
```

```suggestion
      - name: Check if version changed  # Version check based on package.json version number
        uses: EndBug/version-check@v1.6.0
        id: check
```

Id must be provided, even official example has it https://github.com/EndBug/version-check/blob/master/doc/auto-publish-example.yml#L21

otherwise this could not work `steps.check.outputs.changed`